db/migrate/20230301083053_creat_comments.rb

```diff
@@ -0,0 +1,3 @@
+class CreatComments < ActiveRecord::Migration[7.0]
+  def change; end
+end
```

Comment:
- [ ] The migrations and [this one](https://github.com/soesandarwin2201/blog_with_ruby_on_rails/pull/1/files#diff-f9a7b839c2af34d5ecd03e353a175124d7d411427181c41d72727932ec01a4e2) seem to be doing the same thing. Kindly remove one to avoid errors.